### DISCUSSION TITLE
Respon dari GET /messages tidak sesuai format JSON Contoh

### DISCUSSION THREAD

**Discussion by melodiazz** [kirimwa-id/forum#49] [Tanya Jawab]
hi pak @rioastamal 

Mau tanya untuk respon yang saya dapatkan dari GET /messages sepertinya tidak sesuai dengan Format JSON yang dicontohkan.

Contoh yang ditampilkan pada dokumentasi api
![image](https://user-images.githubusercontent.com/22655268/135809815-b52c1f7e-d76b-4791-b904-5710d726a280.png)

Respon yang saya dapatkan dari dari GET /messages

![image](https://user-images.githubusercontent.com/22655268/135809953-5e1cd9a6-ef02-453c-8059-62cbe3c1c340.png)

**Comment by rioastamal** [comment]
@melodiazz respon yang dicontohkan terpotong apa ada yang lebih lengkap dan terformat dengan baik? Bisa di-beautify JSON online dulu biar enak dilihat.

**Reply by rioastamal** [reply]
API KirimWA.id memang tidak melakukan beautify. Di dokumentasi memang sengaja ditampilkan format dengan beautify agar mudah dibaca. JSON diatas dapat dibeautify tanpa masalah misal dengan tools `jq`.

![Screen Shot 2021-10-04 at 16 14 03](https://user-images.githubusercontent.com/469847/135825821-78b082ef-5dbf-4b0c-8c58-7ec66f7143c8.png)

@melodiazz sebaiknya JSON yang diposting dihapus karena sepertinya banyak terdapat informasi sensitif seperti nomor telepon dll.

**Comment by kafaa-dev** [comment]
Format apa ya mas yang beda? Apakah karena ada backslash? atau yang lain?
